lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -159,16 +159,23 @@ function makeComparison(name, actualResult, expectedResult) {
  * @param {Smokehouse.ExpectedRunnerResult} expected
  */
 function pruneExpectations(localConsole, lhr, expected) {
-  const userAgent = lhr.environment.hostUserAgent;
-  const userAgentMatch = /Chrome\/(\d+)/.exec(userAgent); // Chrome/85.0.4174.0
-  if (!userAgentMatch) throw new Error('Could not get chrome version.');
-  const actualChromeVersion = Number(userAgentMatch[1]);
+  /**
+   * Lazily compute the Chrome version because some reports are explicitly asserting error conditions.
```

Comment:
doesn't this mean that the LHR type is wrong for `lhr.environment.hostUserAgent`? Right now it's required even for a 403 page

Reply:
No, the type is right, it's just unknown so the match fails.

The fix there is coming, but didn't want to mingle the two concerns in one PR and still wanted smokes to capture this fix since the 403 doesn't assert the chrome version anyway :)

Reply:
Yeah, fair. I guess ideally the comment will be obsolete soon, though.

Maybe we should add an assertion on the environment/chrome version in one of these smoke tests (maybe in `errors-*` somewhere)? My hesitation is that we're losing a bit of test coverage here because doing that regex _should_ be fine and it's apparently the only thing catching the difference between `''` and a UA string, but it's a very good point that an explicit assertion is way better than a buried, implicit assertion that produces an inscrutable stack trace :)

Reply:
> an explicit assertion is way better than a buried, implicit assertion that produces an inscrutable stack trace :)

Yup, 100%. I've got a proposal for how to tackle this category of base artifact issue in FR where we'll add the explicit assertion back shortly 👍